src/main/java/racing/CarGroup.java

```diff
@@ -0,0 +1,37 @@
+package racing;
+
+import static java.lang.System.*;
+
+import java.util.HashMap;
+
+/**
+ * 자동차 그룹 클래스
+ */
+public class CarGroup {
+    protected HashMap<Integer, Car> cars;
+    public StringBuilder stringBuilder;
+
+    public CarGroup(HashMap<Integer, Car> cars) {
+        this.cars = cars;
+    }
+
+    /**
+     * 자동차 단체 이동 메서드
+     */
+    public void go() {
+        for (int i=0; i<cars.size(); i++) {
+            cars.get(i).move();
+        }
```

Comment:
```suggestion
        for (Car car : cars.values()) {
            car.move();
        }
```

로 표현될 수 있겠네요!

추가로 `List<Car>`로 저장해도 되지 않았을까요?